src/property-filter/__tests__/property-filter.test.tsx

```diff
@@ -718,11 +718,11 @@ describe('property filter parts', () => {
       expect(wrapper.findTokens()!).toHaveLength(1);
       expect(wrapper.findTokens()![0].findLabel().getElement()).toHaveTextContent('string : first');
       // show more
-      act(() => wrapper.findTokenToggle()!.find('button')!.click());
```

Comment:
I updated the test-utils to return the actual button instead of a wrapper. I'll make a dry-run to ensure the change is safe.

Reply:
Verified here: https://github.com/cloudscape-design/components/pull/899